packages/Webkul/Shop/src/Http/Controllers/API/OnepageController.php

```diff
@@ -179,7 +179,7 @@ public function storeOrder()
 
         Cart::deActivateCart();
 
-        session()->flash('order', $order);
+        session()->flash('order', $order->id);
```

Comment:
For better clarity,

~~~php
session()->flash('order_id', $order->id);
~~~

Reply:
adjusted